### ISSUE TITLE
Add warning about .jpg conversion for avatar upload

### ISSUE THREAD

**Issue by dorothy3242** [ppy/osu-web#2884]
Since all uploaded avatars via the new website are forcibly changed to .jpg images, this should be mentioned somewhere here instead of it catching a user trying to use a partially transparent .png avatar off-guard.

**Comment by MegaApplePi** [comment]
Duplicate of #1579

**Comment by dorothy3242** [comment]
Updated to reflect where the issues do not overlap.

**Comment by peppy** [comment]
Conversion only happens when avatar is larger than the specified dimensions.

**Comment by Xinnoh** [comment]
should also mention the maximum size for uploads

**Comment by peppy** [comment]
Related: ppy/osu-web#2894

**Comment by cl8n** [comment]
it doesn't seem necessary after #4589 (fixed what peppy linked above)

**Comment by peppy** [comment]
Sounds fair.
